mteb/benchmarks/benchmarks/rteb_benchmarks.py

```diff
@@ -10,9 +10,9 @@
   year = {2024},
 }"""
 
-RTEB_MAIN = Benchmark(
+RTEB_MAIN = RtebBenchmark(
     name="RTEB(beta)",
-    display_name="RTEB Retrieval Embedding Benchmark",
+    display_name="Multilingual Retrieval",
```

Comment:
If the blogpost is advertising "RTEB", I'd not be able to automatically connect that RTEB and "Multilingual Retrieval" are the same thing.

Reply:
Will go for that

Reply:
Hmm, right, just for all other benchmarks we haven't used the shorthand (we have used a general purpose name which does not require people to know the content)

but RTEB English is probably reasonable so will change it throughout